empire.go

```diff
@@ -708,6 +760,10 @@ func (e *Empire) StreamLogs(app *App, w io.Writer, duration time.Duration) error
 
 // CertsAttach attaches an SSL certificate to the app.
 func (e *Empire) CertsAttach(ctx context.Context, app *App, cert string) error {
+	if err := authorizeApp(ctx, "CertsAttach", app); err != nil {
```

Comment:
I have the same preference as you re: `AddDomain` etc. but the ACL argument is valid.

I would vote for easier to read actions like `AddDomain` vs: `DomainsCreate` and be explicit when defining ACL permissions.